doc/sphinx/developer_manual/contributing_to_the_documentation/important_syntax.md

```diff
@@ -2,6 +2,447 @@
+## Header 2
+
+(header3)=
+### Header 3
+
+:::{code-block} md
+(header1)=
+# Header 1
+
+(header2)=
+## Header 2
+
+(header3)=
+### Header 3
+:::
+
+
+# Links
+Links are written in the following way:
+:::{code-block} md
+[link within this page](title)
+:::
+[link within this page](title)
+
+:::{code-block} md
+[link to other page](part:dev_manual:chap:contribute_to_doc:sec:index)
+:::
+
+[link to other page](part:dev_manual:chap:contribute_to_doc:sec:index)
+
+:::{code-block} md
+[link to external website](https://github.com/GeodynamicWorldBuilder/WorldBuilder)
+:::
+[link to external website](https://github.com/GeodynamicWorldBuilder/WorldBuilder)
+
+# Admonitions
+
+Adding note, warning or and todo's which stand out can be done with the following syntax:
+
+::::{code-block} md
+:::{todo}
+This is a `TODO` admonition.
+:::
+::::
+:::{todo}
+This is a `TODO` admonition.
+:::
+
+::::{code-block} md
+:::{note}
+This is a `note` admonition.
+:::
+::::
+:::{note}
+This is a `note` admonition.
+:::
+
+::::{code-block} md
+:::{tip}
+This is a `tip` admonition.
+:::
+::::
+:::{tip}
+This is a `tip` admonition.
+:::
+
+::::{code-block} md
+:::{seealso}
+This is a `seealso` admonition.
+:::
+::::
+:::{seealso}
+This is a `seealso` admonition.
+:::
+
+::::{code-block} md
+:::{important}
+This is an `important` admonition.
+:::
+::::
+:::{important}
+This is an `important` admonition.
+:::
+
+::::{code-block} md
+:::{warning}
+This is a `warning` admonition.
+:::
+::::
+:::{warning}
+This is a `warning` admonition.
+:::
+
+::::{code-block} md
+:::{attention}
+This is an `attention` admonition.
+:::
+::::
+:::{attention}
+This is an `attention` admonition.
+:::
+
+::::{code-block} md
+:::{danger}
+This is a `danger` admonition.
+:::
+::::
+:::{danger}
+This is a `danger` admonition.
+:::
+
+::::{code-block} md
+:::{error}
+This is an `error` admonition.
+:::
+::::
+:::{error}
+This is an `error` admonition.
+:::
+
+# Figures
+
+Adding figures looks similair to notes and todo's:
+
+::::{code-block} md
+:::{figure} ../../../../doc/sphinx/_static/images/user_manual/basic_starter_tutorial/BST_17.png
+:name: BST_17_synatx
+:alt: Basic Starter Tutorial section 17 highres result. 
+:align: center
+
+Basic Starter Tutorial section 17 high resolution result. This has 8 times the resolution than the grid file above. Note that you can see the Earth's curvature! 
+:::
+::::
+
+:::{figure} ../../../../doc/sphinx/_static/images/user_manual/basic_starter_tutorial/BST_17.png
+:name: BST_17_synatx
+:alt: Basic Starter Tutorial section 17 highres result. 
+:align: center
+
+Basic Starter Tutorial section 17 high resolution result. This has 8 times the resolution than the grid file above. Note that you can see the Earth's curvature! 
+:::
+
+# Code blocks
+
+Making code blocks is similair to making notes or todos'. Instead of `{note}` you need to use `{code-block}` and then the name of the language which the code should be colored by. For example for json, you would use:
```

Comment:
```suggestion
Highlighting code blocks is similar to making notes or todos. Instead of `{note}` you need to use `{code-block}` and then the name of the language which the code should be colored by. For example for json, you would use:
```